mobile-academy/src/main/java/org/motechproject/nms/mobileacademy/domain/course/LessonContentLesson.java

```diff
@@ -0,0 +1,43 @@
+package org.motechproject.nms.mobileacademy.domain.course;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Contains details about actual content files to be played while playing a lesson.
+ */
+@Entity
+public class LessonContentLesson {
+
```

Comment:
Hmmm. Why do we need to maintain consistency with their schema at our data layer? We've changed schema for Kilkari so they're already going to have to deal with change there. Or do you mean you're trying to keep consistency with the hierarchy that they defined for the JSON/API layer?